examples/chef/NEW_CHEF_DEVICES.md

```diff
@@ -0,0 +1,110 @@
+# CHEF devices
+
+Device definitions reside in `examples/chef/devices` as zap and .matter files.
+
+Generally you need to follow these steps to use a new device:
+
+-   Create a zap/matter configuration (forking an existing one is the easiest)
```

Comment:
We could have a "template config" which would be just the root node/ep0 configured. New devices just need to configure ep1 and save it to the device type name

Reply:
Love the idea

Reply:
Will try it to see what the sideffects are. I believe we may not have a ep1 then (or will it be empty?) and we should probably exclude it from being available as a device.

I totally agree that having a specific template rather than "pick an existing device and ensure you remove its remnants" is a great idea. Thanks!

Reply:
Added a template, removed it from zap. tried to edit it to be a good starting template.

We have no matter file for this template, so not really human readable, but hoping it will not be modified often.